src/Router.js

```diff
@@ -27,7 +27,7 @@ const Router = () => {
         <Route path="/profilepage" element={<ProfilePage />} />
         <Route path="/hostpage" element={<HostPage />} />
         <Route path="/test" element={<ModalPromise />} />
-        <Route path="/kakao-login" element={<Login />} />
+        <Route path="/paymentSuccess" element={<PaymentSuccess />}></Route>
```

Comment:
```suggestion
        <Route path="/paymentSuccess" element={<PaymentSuccess />} />
```
self closing 적용해주세요!